components/reward-calculator/index.js

```diff
@@ -378,6 +378,12 @@ const RewardCalculatorPage = () => {
 		}
 	}, [stakingInfo, amount, selectedAccount, apiInstance, ysFees]);
 
+	useEffect(() => {
```

Comment:
this logic will display wrong input amount(total available staking amount) for already active bonded account

Reply:
for example: 
`5E9sK3sMgTtEuD33ZyZ1hmrnrgWR4K55mwAfPxD4bHk84e51` (westend) account has ~2.9 active bonded amount but calculator shows ~3.94 something which is total possible staking amount

need to add check for already active bonded amount available